x/stakeibc/module_ibc.go

```diff
@@ -101,16 +102,25 @@ func (im IBCModule) OnChanOpenAck(
 	switch {
 	case portID == withdrawalAddress:
 		zoneInfo.WithdrawalAccount = &types.ICAAccount{Address: address, Target: types.ICAAccountType_WITHDRAWAL}
-		im.keeper.RatelimitKeeper.AddAddressToWhitelist(ctx, address)
+
 	case portID == feeAddress:
 		zoneInfo.FeeAccount = &types.ICAAccount{Address: address, Target: types.ICAAccountType_FEE}
-		im.keeper.RatelimitKeeper.AddAddressToWhitelist(ctx, address)
+		rewardCollectorAddress := im.keeper.AccountKeeper.GetModuleAccount(ctx, types.RewardCollectorName).GetAddress()
```

Comment:
I like the approach of setting these in zone registration. Let's just remember to set them in our upgrade handler for existing zones :)

Reply:
yup, there's a TODO in upgrades.go for this!